src/xdocs/config_whitespace.xml

```diff
@@ -1060,6 +1068,15 @@ import static java.math.BigInteger.ZERO;
     &lt;property name=&quot;allowLineBreaks&quot; value=&quot;false&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+         If the annotation is between the type and the array, the check will skip:
```

Comment:
"the check will skip" ==> "the check will skip validation for space(s)"

Reply:
fixed